src/api/routers/usersRouter.ts

```diff
@@ -20,14 +20,6 @@ const createUsersRouter = () => {
     '/selfResendInvitation',
     userController.selfResendInvitation.bind(userController)
   );
-  usersRouter.use('/:userId', enrichWithUserFromParams);
-  usersRouter.get('/:userId', userController.getUserById.bind(userController));
```

Comment:
This was unused. The other endpoints were just moved to the `participantUsersRouter`